job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/properties/PropertiesContext.groovy

```diff
@@ -173,4 +173,18 @@ class PropertiesContext extends AbstractExtensibleContext {
             }
         }
     }
+
+    /**
+     * Configures the Gitlab Connection to use
+     *
+     * Gitlab connections can be configured under global settings
+     *
+     * @since 1.46
+     */
+    @RequiresPlugin(id = 'gitlab-plugin', minimumVersion = '1.2.0')
+    void gitlabConnection(String connectionName) {
```

Comment:
remove this amd use the [automatically generated DSL](https://github.com/jenkinsci/job-dsl-plugin/wiki/Automatically-Generated-DSL) for new stuff